### ISSUE TITLE
Improve gestures that lead to account quick switcher modal sheet

### ISSUE THREAD

**Issue by boscojwho** [mlemgroup/mlem#417]
## Introduction
Currently, users can access the account quick switcher in two ways:
1. Long press account tab.
2. Swipe up from anywhere on tab bar.

## Proposals
I'm proposing two changes and one removal:

**Changes**:
1. Account tab icon should indicate ability to quickly switch between multiple accounts.
![IMG_1174](https://github.com/mlemgroup/mlem/assets/2549615/b80a3f3e-4d7e-4ea9-93e2-f801f6689dc7)
![IMG_1173](https://github.com/mlemgroup/mlem/assets/2549615/9fbabf85-3b8e-41e9-8fc6-ab0479840891)
2. If Account tab is already selected, and content is already scrolled up to top, a tap on that tab should open the account quick switcher.

**Remove**:
1. Drag gesture on tab bar that opens account quick switcher. (see issue #414)
- TL;DR: Personally have been seeing the account quick switcher appear when I reopen the app (swiping up on home indicator to exit app is causing app to trigger tab bar drag gesture).

## Discussion
Would love to hear thoughts.

**Comment by EricBAndrews** [comment]
> Account tab icon should indicate ability to quickly switch between multiple accounts.

Could you explain a little of the motivation for this point? I'm not sure I quite see why it's necessary--if it's for feature discoverability we can handle that at onboarding without muddying the tab bar.

**Comment by boscojwho** [comment]
I wouldn't say it's absolutely necessary, more of a possibly nice-to-have, and seems to be a pretty common design pattern for apps that allow multiple user accounts (to be fair, it does look much better when tab bar items only have an icon and no label text, and less so when there's label text).
- Discoverability could become an issue later on if/when more casual users come on board (I'd say discoverability was one of the big issues that killed 3D Touch on iPhone), casual users tend to forget about invisible features. Probably not a problem right now with the relatively engaged "hardcore" audience XD

Definitely low priority in the grand scheme of things.
